libobs/util/platform-nix.c

```diff
@@ -1047,6 +1050,21 @@ uint64_t os_get_proc_virtual_size(void)
 	return (uint64_t)statm.virtual_size;
 }
 #endif
+
+#ifndef __OpenBSD__
+uint64_t os_get_sys_total_size(void)
+{
+	struct sysinfo info;
+	if (sysinfo(&info) < 0)
+		return 0;
+	return (uint64_t)info.totalram * info.mem_unit;
```

Comment:
Prefer a blank line between the `if` body and the following `return` statement, or use braces for the `if` and the `return` can go on the line immediately after the closing brace.